_includes/tutorials/time-concepts/ksql/markup/test/run-tests.adoc

```diff
@@ -0,0 +1,11 @@
+Finally, invoke the tests using the test runner and the statements file that you created earlier:
```

Comment:
We don't use this either. Can we remove it?

Reply:
Removed